### PR TITLE
Tokyo - Anthony

### PR DESCRIPTION
**Pull request by Antsstar**
Changed a few headings and I want to stick to heading font / red-color. Also finished Intro completely

### CODE REVIEW

File: src/css/_section-intro.css [hunk 1]
```diff
@@ -3,3 +3,10 @@
  *
  * Start all rules with .section-intro
  */
+.section-intro .section-heading{
+  font-family: 'Revalia', cursive  ;
```
Comment (wmoussa) [inline src/css/_section-intro.css:7]: hey Anthony, this is just to show you how to leave a comment on a PR. Can you use Times News Font

Comment (Antsstar) [reply src/css/_section-intro.css:7]: testing reactions

File: src/css/_section-intro.css [hunk 2]
```diff
@@ -3,3 +3,10 @@
...
+  color: red
```
Comment (jackarmley) [inline src/css/_section-intro.css]: no semicolon here

File: src/html/_index.html [hunk 3]
```diff
@@ -1,9 +1,8 @@
-<!DOCTYPE html>
 <html>
   <head>
     <!-- @depends _head.html -->
   </head>
-  <body>
+  <body>Anthony
```
Comment (jackarmley) [inline src/html/_index.html]: This shouldn't be here on it's own like this. Maybe remove this?

**Review by jackarmley — CHANGES_REQUESTED** [review]
Review these then you'll be good to go!

**Pull request merged**